src/ontology/mondo-edit.obo

```diff
@@ -338937,7 +338940,7 @@ xref: Orphanet:466703 {source="MONDO:equivalentTo"}
 xref: UMLS:C4225190 {source="MEDGEN:895025", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0005501 {source="DC-OMIM:616829", source="DOID:0070268", source="OMIM:616829"} ! congenital disorder of glycosylation type II
 is_a: MONDO:0017749 {source="Orphanet:466703"} ! disorder of multiple glycosylation
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/18085 {source="MONDO:mim2gene_medgen", source="OMIM:616829"} ! TMEM199
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/18085 {source="MONDO:mim2gene_medgen"} ! TMEM199
```

Comment:
I still see this one on the omim website

Reply:
this should have been maintained `relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/18085 {source="MONDO:mim2gene_medgen", source="OMIM:616829"} ! TMEM199`